parts/k8s/kubernetesparams.t

```diff
@@ -346,45 +346,17 @@
       },
       "type": "string"
     },
-    "kubernetesACIConnectorSpec": {
-      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorSpec"}}
-      "metadata": {
-        "description": "The container spec for ACI Connector."
-      },
-      "type": "string"
-    },
-    "kubernetesACIConnectorClientId": {
-      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorClientId"}}
-      "metadata": {
-        "description": "Client id for ACI Connector."
-      },
-      "type": "string"
-    },
-    "kubernetesACIConnectorClientKey": {
-      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorClientKey"}}
-      "metadata": {
-        "description": "Client key for ACI Connector."
-      },
-      "type": "string"
-    },
-    "kubernetesACIConnectorTenantId": {
-      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorTenantId"}}
-      "metadata": {
-        "description": "Tenant id for ACI Connector."
-      },
-      "type": "string"
-    },
-    "kubernetesACIConnectorSubscriptionId": {
-      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorSubscriptionId"}}
+    "kubernetesACIConnectorEnabled": {
+      {{PopulateClassicModeDefaultValue "kubernetesACIConnectorEnabled"}}
```

Comment:
@sozercan see this PR:

https://github.com/Azure/acs-engine/pull/2915/files

Because this is a "bool" type we can't put a string in there (which is what PopulateClassicModeDefaultValue does)

Reply:
@jackfrancis thanks! updated